src/stravalib/client.py

```diff
@@ -988,9 +991,9 @@ def upload_activity(
     def get_activity_zones(
         self, activity_id: int
     ) -> list[model.BaseActivityZone]:
-        """Gets zones for activity.
+        """Gets activity zones for activity.
```

Comment:
This is where i'm struggling with tests as the test for this has data with floating points within it.